src/doctest_setup.rs

```diff
@@ -160,6 +160,81 @@ cfg_if::cfg_if! {
             create_tables(&mut connection).await;
 
 
+            connection
+        }
+    }  else if #[cfg(feature = "sqlite")] {
+        use diesel_async::sync_connection_wrapper::SyncConnectionWrapper;
+        use diesel::sqlite::SqliteConnection;
+        #[allow(dead_code)]
+        type DB = diesel::sqlite::Sqlite;
+        #[allow(dead_code)]
+        type DbConnection = SyncConnectionWrapper<SqliteConnection>;
+
+        fn database_url() -> String {
+            database_url_from_env("SQLITE_UNIT_TEST_DATABASE_URL")
```

Comment:
```suggestion
            database_url_from_env("SQLITE_DATABASE_URL")
```

(As we don't have the same concurrent schema modification issues as with mysql here)